lms/djangoapps/teams/tests/test_views.py

```diff
@@ -363,7 +363,7 @@ def test_filter_text_search(self):
     @ddt.data(
         (None, 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
         ('name', 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
-        ('open_slots', 200, ['Wind Team', u'sólar team', 'Nuclear Team']),
+        ('open_slots', 200, ['Wind Team', 'Nuclear Team', u'sólar team']),
```

Comment:
It may be worth adding a comment to that effect in the code for future maintainers of this file